qualtran/bloqs/arithmetic/addition.py

```diff
@@ -359,6 +361,63 @@ def _cvs_converter(vv):
     return tuple(int(v) for v in vv)
 
 
+@frozen
+class XorK(Bloq):
```

Comment:
cool! I would probably move this to its own module in e.g. `arithmetic/bitwise.py` or something. but this organization isn't blocking this pr

Reply:
moved to `arithmetic/bitwise.py`